source/dub/internal/utils.d

```diff
@@ -95,6 +97,18 @@ auto lockFile(string path, Duration timeout)
 	}
 }
 
+static this()
```

Comment:
This gets called once per thread, I'm not sure if dub is multithreaded, but either way this only needs to be done once with `shared static this()`.

Reply:
:+1: Updated.